src/node_contextify.cc

```diff
@@ -184,6 +184,32 @@ Local<Context> ContextifyContext::CreateV8Context(
   CHECK(name->IsString());
   Utf8Value name_val(env->isolate(), name);
 
+  Local<Value> codegen = options_obj->Get(env->context(),
+      FIXED_ONE_BYTE_STRING(env->isolate(), "codeGeneration"))
+    .ToLocalChecked();
+
+  if (!codegen->IsUndefined()) {
+    CHECK(codegen->IsObject());
+    Local<Object> codegen_obj = codegen.As<Object>();
+
+    Local<Value> allowcgfs = codegen_obj->Get(env->context(),
+        FIXED_ONE_BYTE_STRING(env->isolate(), "strings"))
+      .ToLocalChecked();
+    if (!allowcgfs->IsUndefined()) {
+      CHECK(allowcgfs->IsBoolean());
+      ctx->AllowCodeGenerationFromStrings(allowcgfs.As<Boolean>()->Value());
+    }
+
+    Local<Value> allowwcg = codegen_obj->Get(env->context(),
+        FIXED_ONE_BYTE_STRING(env->isolate(), "wasm"))
+      .ToLocalChecked();
+    if (!allowwcg->IsUndefined()) {
+      CHECK(allowwcg->IsBoolean());
+      if(allowwcg.As<Boolean>()->Value() == false)
+        ctx->SetEmbedderData(kAllowWasmCodeGeneration, True(env->isolate()));
+    }
```

Comment:
Apropos the variable names, `allow_wasm_code_generation` is clearer than `allowwcg`.